src/inspect_ai/log/_log.py

```diff
@@ -690,6 +690,9 @@ class EvalRevision(BaseModel):
     commit: str
     """Revision commit."""
 
+    dirty: bool = False
```

Comment:
Oh sorry, this is where you need the Field designator (that's our style for defaults rather than just a literal)